packages/rocketchat-ui-message/message/renderMessageBody.js

```diff
@@ -22,5 +22,46 @@ renderMessageBody = function(msg) {
 
 	msg.html = message.html.replace(/\n/gm, '<br/>');
 
+	if (singleEmojiCheck(msg)) {
+		msg.html = (msg.html).replace('emojione ', 'big_emojione ');
+	}
+
 	return msg.html;
 };
+
+/**
+ *Checks to see if a message contains only an emoji span
+ *@para {msg} some html message
+ *@return {false} the html message either doesn't 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2488056950/issues/source?bid=3199634&fileBranchId=3266617#l35)